k2/csrc/array_test.cu

```diff
@@ -108,6 +108,25 @@ void TestArray1() {
     }
   }
 
+  {
+    // created with Array1(ContextPtr, int32_t size, Callable &&callable)
+    auto lambda_set_values = [=] __host__ __device__(int32_t i) -> T {
```

Comment:
This lambda does not need to capture anything.

Reply:
Fixed.